diff --git a/src/Makefile b/src/Makefile
index b733a11..467310b 100644
--- a/src/Makefile
+++ b/src/Makefile
@@ -142,6 +142,10 @@ tests: blst $(SOURCE_FILES) $(HEADER_FILES)
 	@echo "[+] building tests"
 	@$(CC) $(CFLAGS) -o $@ test/tests.c $(LIBS)

+bc: blst $(SOURCE_FILES) $(HEADER_FILES)
+	@echo "[+] generating bitcode"
+	@$(CC) $(CFLAGS) -c -emit-llvm ckzg.c
+
 # This simply runs the test suite.
 .PHONY: test
 test: tests
